include/MySQL_Monitor.hpp

```diff
@@ -127,6 +131,16 @@ class AWS_Aurora_monitor_node {
 	}
 };
 
+class AWS_RDS_topology_server {
```

Comment:
Since this has no internal logic, I don't see a strong motivation for keeping it as a class instead of a simple struct with default initialized fields, this way there is no need to define a default ctor.